docs/computer_setup.md

```diff
@@ -2,82 +2,161 @@
-	* pwd: print working directory you are in. If you want to know the exact directory you are working within, then pwd will tell you. 
-	* cd: change directory you are in.
-	* exit: exit out of the directory. This command will close a terminal window, and end the execution of a shell script. 
+	- Here are some commonly used commands in the Bash terminal:
+	- `ls`: list directory contents. The `ls` command allows you to quickly view all
+	files within your current working directory (just run `ls`) or any specified
+	directory (run `ls <path_name>` by replacing `<path_name>` with the path you are
+	interested in).
+	- `pwd`: print working directory. This prints the full path to the current
+	directory you are working in. 
+	- `cd`: change directory. If you run this by itself, it will move you to your "home"
+	directory. If you pass either a relative or full path to it (`cd <path_name>`) you
+	will move to that path. You can confirm that you moved by running `pwd` and you can
+	see what is inside the folder you are currently in by running `ls`.
 
-* Git
-	Git is a version control system that lets you track who made changes to what and when on github.com. It also makes collaboration easier, allowing changes by multiple people to all be merged into one source. You can access the git via command line (terminal) or a desktop app that has a GUI, such as GitKraken. 
-
-* GitHub
-	* Github is a software that runs locally on your computer and your files and their history are stored on your computer. Having a centrally located place to upload files and make changes and download changes from others enables you to collaborate more easily with other developers. 
+- Git
+	Git is a version control system that lets you track who made changes to what files
+	and at what time. A folder of code that is tracked together by git is called a
+	repository. Git runs locally on your computer and keeps track of the changes you
+	make and allows you to see all the past changes you and others have made. It also
+	makes collaboration easier by making it easy to coordinate changes by multiple
+	people. You can access the git via command line (terminal) or a desktop app that
+	has a GUI, such as GitKraken (strongly recommended!). 
 
+- GitHub
+	- Github is a website that provides hosting of git repositories and adds some very
+	helpful collaboration tools as well. It provides a centrally located, always online,
+	access point for git repositories.
 	
-* Python 
-	* Python is a high-level, general-purpose, interpreted computer programming language. It is widely applied in many fields to build solutions for web applications, software development, data analysis and conducting AI techniques. Python has two major versions: 2x was released in 2000 and the latest version 3x was released in 2008. It’s recommended to use the latest version 3 in projects.
+- Python 
+	- Python is a high-level, general-purpose, interpreted computer programming
```

Comment:
These terms don't tend to mean much to non-programmers. Consider instead stating the effects these terms are implying: "Python is a programming language that prioritizes being human-readable, adaptable to many kinds of problems, and easy to write quickly without careful study."